src/java.base/share/classes/java/text/DateFormat.java

```diff
@@ -117,7 +117,7 @@
  * different options to these factory methods to control the length of the
  * result; from {@link #SHORT} to {@link #MEDIUM} to {@link #LONG} to {@link #FULL}. The exact result depends
  * on the locale, but generally:
- * <ul><li>{@link #SHORT} is completely numeric, such as {@code 12.13.52} or {@code 3:30pm}
+ * <ul><li>{@link #SHORT} is the shortest and mainly numeric, such as {@code 12.13.52} or {@code 3:30pm}
```

Comment:
Not sure if we want to update the descriptions of all options, but if we want to be consistent, here's a suggestion, how about:
  {@link #SHORT} is a concise format using numeric digits, for example, {@code 12.13.52} or {@code 3:30pm}
  {@link #MEDIUM} provides more detail, such as {@code Jan 12, 1952}
  {@link #LONG} is a comprehensive format, such as {@code January 12, 1952} or {@code 3:30:32pm}
  {@link #FULL} provides a complete date representation, such as
  
  "pretty completely" is a bit casual :-)

Reply:
Thanks for the feedback, Joe. I think I would rather keep the description somewhat abstract, simply mentioning the formatted output is `SHORT` <= `MEDIUM` <= `LONG` <= `FULL` in length-wise. As it is noted in the previous sentence, it is pretty much locale-dependent.
I simply replaced that casual wording with "the longest" for consistency.

Reply:
Added a sentence explaining it.